engine/src/render/g_buffer/plugin.cpp

```diff
@@ -0,0 +1,53 @@
+#include <cubos/core/io/window.hpp>
+#include <cubos/core/reflection/external/primitives.hpp>
+
+#include <cubos/engine/render/g_buffer/g_buffer.hpp>
+#include <cubos/engine/render/g_buffer/plugin.hpp>
+#include <cubos/engine/render/target/target.hpp>
+
+using cubos::core::gl::Texture2DDesc;
+using cubos::core::gl::TextureFormat;
+using cubos::core::gl::Usage;
+using cubos::core::io::Window;
+
+void cubos::engine::gBufferPlugin(Cubos& cubos)
+{
+    cubos.addComponent<GBuffer>();
+
+    cubos.tag("CreateGBufferTag");
```

Comment:
Tag naming looks different from usual. Does this have anything to do with the upcoming typed tags?

Reply:
Yeah, kept it like that so that @joaomanita knows what it should be named like if we merge this first.